modules/ROOT/pages/hot-deployment.adoc

```diff
@@ -0,0 +1,30 @@
+= Application Hot Deployment
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: runtime, deploy, hot deployment, application, applications
+
+Modify your configuration files and custom classes and reload them without having to restart Mule.
+
+== Outline of a Deployment
+
+Here is a quick summary for deploying an app 'foo':
+
+* Create a directory under `$MULE_HOME/apps/foo`
+* Place your packaged application (e.g. `foo.jar`) at `$MULE_HOME/apps`
+* Compress to a jar custom classes (if any), and put them under `$MULE_HOME/apps/foo/META-INF/mule-src/foo/src/main/java/`
+* Start your app with `mule -M-Dmule.deploy.applications=foo`
+
+As a bonus, application's configuration files are monitored, so if there are any class changes you want to pick up or simply modify any config, save or touch any of the Mule configuration files and Mule will hot-reload the application.
```

Comment:
I would leave out "as a bonus" even though I'm sure it's a bonus.  :-)
The application's configuration files are monitored, which means that if there are any class changes you want to add, or changes to the config, you can save or touch any of the Mule configuration files, and Mule will hot reload the application.